proto/cosmwasm/slpp/v1/slpp.proto

```diff
@@ -9,16 +9,18 @@ service Msg {
 
 // AVS represents the state-ful information stored per AVS
 message AVS {
-    string contract_address = 1;
+  bytes contract_bin = 1;
```

Comment:
Feel like the identifier here should just be code id, and the code id in the wasm module is what stores the bytes

Reply:
I'm a little unsure of where we left off on the conversation, but my understanding was a contract could potentially be associated with more than one AVS